templates/configfile.j2

```diff
@@ -59,7 +59,7 @@ checkmnt=
 sapmnt={{ sap_hana_deployment_hana_install_path }}
 
 # Local Host Name
-hostname={{ ansible_hostname }}
+hostname={{ sap_hana_deployment_hostname|default(ansible_hostname) }}
```

Comment:
I can't see `sap_hana_deployment_hostname` defined anywhere or as part of the README

Reply:
Added